cuda/spotfinder/spotfinder.cc

```diff
@@ -284,7 +284,7 @@ float get_resolution(int order,
                      float wavelength,
                      float distance_to_detector,
                      float distance_from_center) {
-    float theta = atan(distance_from_center / distance_to_detector);
+    float theta = 2 * atan(distance_from_center / distance_to_detector);
```

Comment:
Sorry but I think it's the other way round, the angle in the triangle is two*theta, so it should be theta = 0.5 * atan (where theta is the theta in Braggs equation)?

Reply:
You are absolutely right, I put it the wrong way around. I'll sort that now.